storm-core/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -1338,6 +1339,26 @@ public static void handleUncaughtException(Throwable t) {
         }
     }
 
+    public static void validateTopologyBlobStoreMap(Map stormConf, Set<String> blobStoreKeys) throws InvalidTopologyException {
+        boolean containsAllBlobs = true;
```

Comment:
Minor: We're adding missing keys to Set so no need to have additional flag. `missingKeys.size() > 0` is same to `containsAllBlobs == false`.